Tests/ComposableArchitectureTests/MacroTests.swift

```diff
@@ -55,28 +55,28 @@
     }
   }
 
-//  enum TestEnumReducer_Basics {
-//    @Reducer struct Feature {}
-//    @Reducer
-//    enum Destination {
-//      case feature(Feature)
-//    }
-//  }
+  enum TestEnumReducer_Basics {
```

Comment:
These tests were failing on CI before the enum-reducers branch was merged, and in haste I just commented them out because I thought it was due to something else. Well, turns out it was catching the problem this PR fixes. So now we can bring them back.